lib/tokenize.cpp

```diff
@@ -9093,6 +9093,8 @@ void Tokenizer::simplifyNamespaceStd()
 
     for (const Token* tok = Token::findsimplematch(list.front(), "using namespace std ;"); tok; tok = tok->next()) {
         bool insert = false;
+        if (Token::Match(tok, "enum class|struct| %name%| :|{")) // Don't replace within enum definitions
+            tok = Token::findsimplematch(tok, ";");
```

Comment:
I would have also searched for the { first and if that is found then jump to the }